src/routes/home.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  GoAButton,
+  GoAButtonGroup,
+  GoADivider,
+  GoAGrid,
+  GoAHeroBanner,
+  GoAHeroBannerActions,
+  GoAPageBlock,
+} from "@abgov/react-components";
+import "./home.css";
+import { MAX_CONTENT_WIDTH } from "@routes/root";
+import CardLite from "../components/card-lite/CardLite";
+import { ContactSupport } from "@components/contact-support/ContactSupport";
+import { Link, useNavigate } from "react-router-dom";
+const HomePage = () => {
+  const navigate = useNavigate();
+  return (
+    <>
+      <GoAHeroBanner
+        heading="Design your service using reusable guidelines, tokens, components,
+        and patterns."
+        maxContentWidth={MAX_CONTENT_WIDTH}
+      >
+        Use these resources to build your service. Use and learn from the
+        research and experience of other service teams and avoid repeating work
+        that's already been done.
+        <GoAHeroBannerActions>
+          <GoAButtonGroup alignment="start">
+            <GoAButton
+              onClick={() => {
+                navigate("get-started");
+              }}
+              trailingIcon="arrow-forward"
+            >
+              Get Started
+            </GoAButton>
+            <GoAButton onClick={() => {}} type="tertiary">
+              Learn about v1.5.0
+            </GoAButton>
+          </GoAButtonGroup>
+        </GoAHeroBannerActions>
+      </GoAHeroBanner>
+
+      <GoAPageBlock width={MAX_CONTENT_WIDTH}>
+        <GoAGrid minChildWidth="300px" mt="2xl" mb="2xl" gap="2xl">
+          <CardLite
+            title="Patterns and templates"
+            description="Patterns and page templates to create a better user experience for government services."
+            linkTo="/patterns"
+            linkDisplay="Browse patterns"
+          />
+          <CardLite
+            title="Components"
+            description="Reusable parts of the user interface such as buttons, modals, steppers, and text inputs."
+            linkTo="/components"
+            linkDisplay="Browse Components"
+          />
+          <CardLite
+            title="Styles"
+            description="Use the design system styles within your service, shared across design and development."
+            linkTo="/design-tokens"
+            linkDisplay="Browse Styles"
+          />
+          <CardLite
+            title="Content"
+            description="Use the design system styles within your service, shared across design and development."
+            linkTo="/content"
+            linkDisplay="Browse Content"
+          />
+        </GoAGrid>
+
+        <GoADivider mb="2xl"></GoADivider>
+        <div className="large-text">
```

Comment:
This doesn't match the UI specs. The "Remember" section in your code has a small font.

![image](https://github.com/GovAlta/ui-components-docs/assets/41582/8a2af885-6299-4fda-8ffe-78e4fe10863d)

![image](https://github.com/GovAlta/ui-components-docs/assets/41582/974738ce-f227-41ee-9d79-f543747b510d)

Reply:
Also the `large-text` class wraps the `h2` tag, which makes things a little confusing. If you can move the `h2` out that would be better.

Reply:
The class class name of `large-text` is not ideal. Currently in how things are set up all css classes are global, so generic names may result in unintentional overlap. Renaming this css class to a more semantic name would be ideal ex. `home__using-design-system` or something like that.